src/ui_utils.c

```diff
@@ -2477,10 +2477,24 @@ void ui_init_builder(void)
 static void init_custom_style(void)
 {
 #if GTK_CHECK_VERSION(3, 0, 0)
-	gchar *css_file = g_build_filename(app->datadir, "geany.css", NULL);
+	const struct {
+		guint version;
+		const gchar *file;
+	} css_files[] = {
+		{ 20, "geany-3.20.css" },
+		{ 0, "geany-3.0.css" },
```

Comment:
It might also be worth putting a comment on these initializers telling that they must be in descending order and that the one with 0 for the `version` must be last, as eluded to in the `while` loop comment below.